sdk/python/foundation-models/system/inference/automatic-speech-recognition/asr-batch-endpoint.ipynb

```diff
@@ -164,7 +163,7 @@
    "outputs": [],
    "source": [
     "model_name = \"openai-whisper-large\"\n",
-    "model_version = \"4\"\n",
+    "model_version = \"8\"\n",
```

Comment:
model version 8 or 9?

Reply:
I will update to 9 once the release is completed, is that fine?

Reply:
@tanmaybansal104